README.rst

```diff
@@ -177,3 +177,11 @@ PRAW's source (v4.0.0+) is provided under the `Simplified BSD License
 
 Earlier versions of PRAW were released under `GPLv3
 <https://github.com/praw-dev/praw/blob/0c88697fdc26e75f87b68e2feb11e101e90ce215/COPYING>`_.
+
+Sponsors
+--------
+
+.. image:: docs/package_info/NucleiLogo.png
```

Comment:
We might have to give an absolute URL on GitHub (relative to the commit) for this to work in other places that the README shows up.